src/sayfudinova/task2/CubeWindow.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Base/GLWindow.hpp>
+#include <QColor>
+#include <QColorDialog>
+#include <QKeyEvent>
+#include <QMatrix4x4>
+#include <QOpenGLBuffer>
+#include <QOpenGLShaderProgram>
+#include <QQuaternion>
+#include <QVector2D>
+#include <QVector3D>
+#include <memory>
+#include <chrono>
+
+class CubeWindow final : public fgl::GLWindow {
+
+public:
+  void init() override;
+  void render() override;
+
+protected:
+  void mousePressEvent(QMouseEvent *e) override;
```

Comment:
Давайте для единообразия (и ниже)
```suggestion
  void mousePressEvent(QMouseEvent *event) override;
```